test/pstl_offload/headers/headers.standard/algorithms/algorithm.pass.cpp

```diff
@@ -0,0 +1,17 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <algorithm>
```

Comment:
I think we should not, because we don't guarantee that any oneDPL API would be available after including the standard headers. We only guarantee that we offload would work somohow.